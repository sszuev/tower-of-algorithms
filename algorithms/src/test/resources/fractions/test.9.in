1/9973+1/9999