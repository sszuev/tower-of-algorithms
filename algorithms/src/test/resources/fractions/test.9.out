19972/99720027